packages/react/runtime/__test__/typecheck/jsx-runtime.test-d.tsx

```diff
@@ -0,0 +1,104 @@
+// Copyright 2024 The Lynx Authors. All rights reserved.
```

Comment:
You would also need to exclude the coverage of this

https://github.com/lynx-family/lynx-stack/blob/64f64ddc6f61bb9091fee57f5e8b5ba0efed77f1/packages/react/runtime/vitest.config.ts#L71

https://github.com/lynx-family/lynx-stack/blob/64f64ddc6f61bb9091fee57f5e8b5ba0efed77f1/vitest.config.ts#L11